src/main/menus/app.js

```diff
@@ -186,14 +186,14 @@ function createTemplate(mainWindow, config, isDev) {
       };
     }), separatorItem, {
       label: 'Select Next Server',
-      accelerator: (process.platform === 'darwin') ? 'Alt+Cmd+Right' : 'CmdOrCtrl+Tab',
+      accelerator: 'Ctrl+Tab',
```

Comment:
I had thought `Alt+Cmd+Right` is appropriate here (See Chrome's menu), but Safari uses `Ctrl+Tab`. I think it's more consistent on macOS 👍